tests/recognizer-tests.ts

```diff
@@ -24,6 +24,14 @@ QUnit.test("A simple route recognizes", (assert: Assert) => {
   assert.equal(router.recognize("/foo/baz"), null);
 });
 
+QUnit.test("A simple route with double trailing slashes recognizes", (assert: Assert) => {
```

Comment:
If recognize should clean these, why is it only important to clean the trailing double slashes?